src/System.Net.ServicePoint/tests/System.Net.ServicePoint.Tests.csproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
```

Comment:
Thanks.  I just copied the project from System.Net.Mail, which has the same issue:
https://github.com/dotnet/corefx/blob/master/src/System.Net.Mail/tests/Functional/System.Net.Mail.Functional.Tests.csproj#L6
I'll fix it there as well.